modules/core/src/layers/xviz-layer.js

```diff
@@ -400,6 +448,21 @@ export default class XVIZLayer extends CompositeLayer {
           })
         );
 
+      case XVIZ_PRIMITIVE_TYPES.image:
```

Comment:
What does this constant buy us?  The rest of the code is just using a string.  Just curious if i'm missing something.
Mainly this creates a dissonance in the code and prompts the question "why"